docs/requirements.txt

```diff
@@ -1,2 +1,3 @@
 # Force rtfd to use a recent version of mkdocs
 mkdocs==1.4.2
+pymdown-extensions==10.1
```

Comment:
I think this can be put in pyproject.toml actually now, as RTFD is now using a modern version of mkdocs, and is it now, it is not installed when doing a `make develop` ;)

Reply:
Ah, right, it will be in october, my bad!